pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -2183,16 +2183,13 @@ func TestHandlePodCreationError(t *testing.T) {
 		taskLister:        testAssets.Informers.Task.Lister(),
 		clusterTaskLister: testAssets.Informers.ClusterTask.Lister(),
 		resourceLister:    testAssets.Informers.PipelineResource.Lister(),
-		timeoutHandler:    timeout.NewHandler(ctx.Done(), testAssets.Logger),
+		snooze:            func(acc kmeta.Accessor, amnt time.Duration) {},
```

Comment:
We should implement snooze here either to prove it is called, or prove it isn't, to aid future improvements in this code.

Reply:
good idea, done.